.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: java
 jdk:
   - oraclejdk8
-after_success:
-  - bash .travis_after_success.sh
+#after_success:
+#  - bash .travis_after_success.sh
```

Comment:
Just to make CI work on my branch. As mentioned in the comments, changes to this file can be ignored.